src/core/manager.c

```diff
@@ -3989,9 +3990,12 @@ static int manager_run_generators(Manager *m) {
                 goto finish;
         }
 
-        r = safe_fork("(sd-gens)",
-                      FORK_RESET_SIGNALS | FORK_WAIT | FORK_NEW_MOUNTNS | FORK_MOUNTNS_SLAVE | FORK_PRIVATE_TMP,
-                      NULL);
+        /* On some systems /tmp/ doesn't exist, and on some other systems we cannot create it at all. Avoid
+         * trying to mount a private tmpfs on it as there's no one size fits all. */
+        if (is_dir("/tmp", /* follow= */ true) > 0)
```

Comment:
If `FORK_PRIVATE_TMP` is set, `mount_nofollow()` is used to mount tmpfs on /tmp, hence, `/tmp` cannot be a symlink.
So, please set `false` for `follow`.